src/Xamarin.Android.Tools.Bytecode/ClassPath.cs

```diff
@@ -340,7 +340,6 @@ public void SaveXmlDescription (TextWriter textWriter)
 			var contents    = ToXElement ();
 			using (var writer = XmlWriter.Create (textWriter, settings))
 				contents.Save (writer);
-			textWriter.WriteLine ();
```

Comment:
Why remove this?

The reason for this `WriteLine()` call is for `class-parse` command-line use, e.g.

```bash
$ mono bin/Debug/class-parse.exe bin/Debug/java-interop.jar
<api
...
</api>
```

Without this `WriteLine()` call, there won't be a trailing newline, which is *weird*:

```
</api>jon@localhost:Java.Interop$ 
```

Plus, it's good Unix hygiene to have a trailing newline for text files.

Reply:
Yeah, I did this because it was `\r\n` on Windows and the tests failed.

It might be better to just call `Trim()` in the tests.